oauth2/src/test/groovy/com/okta/spring/boot/oauth/AutoConfigConditionalTest.groovy

```diff
@@ -501,8 +501,7 @@ class AutoConfigConditionalTest implements HttpMock {
         reactiveContextRunner().withPropertyValues(
             "okta.oauth2.issuer=https://test.example.com/oauth2/custom-as",
             "spring.security.oauth2.client.provider.okta.issuerUri=${mockBaseUrl()}oauth2/custom-as", // work around to not validate the https url
-            "okta.oauth2.client-id=test-client-id",
-            "okta.oauth2.client-secret=test-client-secret")
+            "okta.oauth2.client-id=test-client-id")
```

Comment:
The existing tests _should_ function the same way to ensure backward compatibility with the previous version
NOTE: It is possible the tests make improper assumptions though, and will need to be updated